UnityProject/Assets/Scripts/SpawnPointListItem.cs

```diff
@@ -0,0 +1,43 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+namespace SpawnPointSelector
+{
+  public class SpawnPointListItem : MonoBehaviour
+  {
+    public GameObject LeftArrow {
+      get => _leftArrow;
+      set => _leftArrow = value;
+    }
+
+    public GameObject RightArrow {
+      get => _rightArrow;
+      set => _rightArrow = value;
+    }
+
+    public Text Text {
+      get => _text;
+      set => _text = value;
+    }

```

Comment:
I think you can just do {get; set;} and it will generate exactly these getters and setters

Reply:
Can I annotate the property itself with [SerializeField] and have it work?

Reply:
nevermind I am wrong